### PR TITLE
Add Nepali README

### PR DESCRIPTION
**Pull request by nitishkaf**

### CODE REVIEW

File: README.np.md [hunk 1]
```diff
@@ -0,0 +1,118 @@
...
+
+&nbsp;      | &nbsp;
+:---------- | :-----
+कॉपीराइट      | पब्लिक डोमेन
+Comanche055 | कोलोसस २ ए को लागी स्रोत कोड को अंश, कमाण्ड मोड्युल (सीएम) अपोलो गाइडन्स कम्प्युटर (AGC) अपोलो ११ को लागी <br>`AGC प्रोग्राम Comanche को NASA द्वारा संशोधन ० 055 सम्मिलित`<br>`2021113-051. 10:28 APR. 1, 1969`
+Luminary099 |  ल्यूमिनरी १ए, अपोलो ११ के चंद्र भाग (LM) के लिए निर्देशरण संगड़क यंत्र (AGC) के नियमसंग्रह का हिस्सा। <br>`Luminary 1A को लागि स्रोत कोडको भाग, चंद्र मोड्युलको (LM) अपोलो गाइडन्स कम्प्युटर (AGC) अपोलो ११ को लागि`<br>`2021112-061. 16:27 JUL. 14, 1969`
+Assembler     | yaYUL
+संपर्क     | रोन बुर्की (Ron Burkey) <info@sandroid.org>
+वेबसाइट      | www.ibiblio.org/apollo
+डिजिटलीकरण   | यो स्रोत कोड ट्रान्सक्रिप्ट गरिएको छ वा अन्यथा एमआईटी संग्रहालयबाट हार्डकपीको डिजिटलाइज्ड छविहरूबाट रूपान्तरण गरिएको छ। डिजिटलाइजेशन फिल फेजेल(Paul Fjeld)द्वारा गरिएको थियो, र संग्रहालयको डेबोरा डगलस (Deborah Douglas) द्वारा व्यवस्था गरिएको थियो। दुबैलाई धेरै धन्यवाद।
```
Comment (wopian) [inline README.np.md:81]: It's preferred to not transliterate people names on this repo

File: README.np.md [hunk 2]
```diff
@@ -0,0 +1,118 @@
+# अपोलो -11
```
Comment (wopian) [inline README.np.md:1]: Should there be a space between `अपोलो` and `-11`? The dash is not required and can also be written as `अपोलो ११` potentially

**Review by wopian — CHANGES_REQUESTED** [review]

File: README.np.md (hunk 1 above)
Comment (kaflenitish) [reply README.np.md:81]: Got it

**Comment by wopian** [comment]
@kaflenitish this PR is waiting on the changes requested above to be made. Once those have been done it's ready to be merged 👍

**Comment by kaflenitish** [comment]
Thanks! I'm from Nepal and thought my native language added could help fellow Nepalese!

**Pull request merged**